src/main/java/core/basesyntax/strategy/QuantityHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface QuantityHandler {
```

Comment:
```suggestion
public interface OperationHandler {
```
the naming of implementations should be changed respectively